x-pack/test_serverless/functional/test_suites/security/ml/data_frame_analytics_jobs_list.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const svlMml = getService('svlMl');
```

Comment:
typo 
```suggestion
  const svlMl = getService('svlMl');
```

Reply:
Updated in https://github.com/elastic/kibana/pull/167493/commits/6d829cc930afd756b5bf1b02c03097a92cd55883